src/manifest.json

```
@@ -34,7 +34,7 @@
   },
   "web_accessible_resources": [
     {
-      "resources": ["polyfill/*"],
+      "resources": ["polyfill/*", "pages/progress-connect/*"],
```

Doesn't this work as well?
```suggestion
      "resources": ["polyfill/*", "pages/**/*"],
```

---

All pages might need not be web_accessible. For example, the post-install page doesn't need to be.